daemon/daemon_test.go

```diff
@@ -77,19 +77,17 @@ type DaemonSuite struct {
 	OnClearPolicyConsumers    func(id uint16) *sync.WaitGroup
 }
 
-func (ds *DaemonSuite) SetUpTest(c *C) {
+func TestMain(m *testing.M) {
```

Comment:
Is `TestMain` something called by GoCheck? (probably?)

Reply:
Yeah, I find it a bit weird that we are mixing up `go-check` and the default testing library in Golang here.

@raybejjani re: `TestMain` - https://golang.org/pkg/testing/#hdr-Main

Reply:
ah, TIL... this answers my other comment too, I think!

Reply:
@ianvernon `go-check` is explicitly built as an extension to the default `testing` library. Hence it is OK in my book to use `testing` functionality when `go-check` does not provide similar functionality. In this case it is about simply running global initialization code once before any testing is done. I like `TestMain()` better than, e.g., using `sync.Once` in, say, `SetUpSuite()`.